src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -78,17 +82,28 @@
         # with s.group('sql db restore-point') as c:
         #     c.command('list', 'list_restore_points')
 
-        # TDE will not be included in the first batch of GA commands
-        # with s.group('sql db transparent-data-encryption') as c:
-        #     c.command('create', 'create_or_update_transparent_data_encryption_configuration')
-        #     c.command('show-configuration', 'get_transparent_data_encryption_configuration')
-        #     c.command('show-activity', 'list_transparent_data_encryption_activity')
-
         # Service tier advisor will not be included in the first batch of GA commands
         # with s.group('sql db service-tier-advisor') as c:
         #     c.command('list', 'list_service_tier_advisors')
         #     c.command('show', 'get_service_tier_advisor')
 
+    transparent_data_encryptions_operations = create_service_adapter(
+        'azure.mgmt.sql.operations.transparent_data_encryptions_operations',
+        'TransparentDataEncryptionsOperations')
+    with ServiceGroup(__name__, get_sql_database_transparent_data_encryptions_operations,
+                      transparent_data_encryptions_operations, custom_path) as s:
+        with s.group('sql db tde') as c:
+            c.command('update', 'create_or_update')
```

Comment:
Update has a very specific meaning in CLI 2.0.  Since this is likely a "create or replace" style PUT, this command should be called "set".

Reply:
Ok, makes sense but I didn't get this feedback when I added `sql db threat-policy update` or `sql db audit-policy update`. Those both have optional updateable properties thought, so I still think `update` using `c.generic_update_command` makes sense for them. Thoughts?

Reply:
I'm fine with that.  The key thing is that update implies "I update what I want and everything else stays the same as it was."  If I use "set" that implies that whatever I don't specify is null.  So if the command preserves the existing values (which generic update definitely does), then update is definitely the right choice.